proposals/gc/MVP.md

```diff
@@ -576,6 +576,55 @@ In particular, `ref.null` is typed as before, despite the introduction of `none`
   - `array.len : [(ref null array)] -> [i32]`
   - traps on `null`
 
+* `array.fill` fills a slice of an array with a given value
+  - `array.fill $t : [(ref null $t), i32, t, i32] -> []`
+    - iff `expand($t) = array (mut t')`
+    - and `t = unpacked(t')`
+  - the 1st operand is the `array` to fill
+  - the 2nd operand is the `offset` into the array at which to begin filling
+  - the 3rd operand is the `value` with which to fill
+  - the 4th operand is the `size` of the filled slice
+  - traps if `array` is null or `offset + size > len(array)`
```

Comment:
It should also trap if the offset or size are negative afaik. (Similar comment for the operations below.)